test/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQListenerTests.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.Azure.WebJobs.Host.Protocols;
 using Microsoft.Azure.WebJobs.Host.Scale;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;

```

Comment:
1. Add unit tests that verifies defaults
2. Add unit test that verifies values overridden
3. Not sure if we have a Samples E2E that show how to configure RabbitMq options - please do add or update sample that configures these values.

Reply:
Instead of adding it in Samples, I added another test that uses webJobsBuilder with the right PrefetchCount and verifies that the setting returns the same value.